### DISCUSSION TITLE
Is it safe to sign into YouTube music? Is there any bugs for this app? I want to check before signing in?

### DISCUSSION THREAD

**Discussion by horyzen1** [pear-devs/pear-desktop#881] [Q&A] [answered]
Is it safe to sign into YouTube music? Is there any bugs for this app? I want to check before signing in. 

Thanks

**Comment by horyzen1** [comment, marked as answer]
ok never mind there's no bugs

**Comment by inson1** [comment]
@horyzen1 or @th-ch May you close this discussion if its done? :)

**Reply by horyzen1** [reply]
close it, it's done
